packages/liquidator/index.js

```diff
@@ -190,14 +192,31 @@ async function run({
 
     // Create a execution loop that will run indefinitely (or yield early if in serverless mode)
     for (;;) {
+      // Check if EMP is expired.
+      const [expirationTimestamp, contractTimestamp] = await Promise.all([
```

Comment:
@mrice32 here I will set isExpired if it expires during the loop

Reply:
nit: does this shadow the external variables? Could you do this?
```
      [expirationTimestamp, contractTimestamp] = await Promise.all([
        emp.methods.expirationTimestamp().call(),
        emp.methods.getCurrentTime().call()
      ]);

Reply:
Yeah you're right, but I ended up stripping out this logic into its own arrow function